storage/integration_test.go

```diff
@@ -4171,77 +4171,17 @@ func TestIntegration_PostPolicyV4(t *testing.T) {
 	}
 
 	objectName := "my-object.txt"
```

Comment:
Make the object name specific to the test; no reason to cause rate limit issues by reusing.

Reply:
each test is on a different bucket, but doesn't hurt to change the object name as well